pkg/api/validate/api.go

```diff
@@ -63,16 +69,12 @@ func (v *APIValidator) validateUpdateContainerService(cs, oldCs *api.OpenShiftMa
 			}
 		}
 	}
+	old.Properties.AuthProfile.IdentityProviders = cs.Properties.AuthProfile.IdentityProviders
```

Comment:
@ehashman and @mjudeikis here we are copying cs to old, so the 2 can't be different. So there is no point in having the redact - as far as i can see.

Reply:
I think redact was for sync pod not to log secretes into the logs if those are changed. If those are not same, sync used to print diff values. I think this is not solved by this change?

Reply:
@mjudeikis given that we are now coping cs to old, it is impossible for it to print out.
remember this was to print out properties that were invalid to change, since we are now saying that you can change this, this won't print.